stream-portfolio/portfolio-core/src/main/java/com/backbase/stream/portfolio/service/PortfolioIntegrationService.java

```diff
@@ -140,7 +140,7 @@ public Mono<Position> upsertPosition(Position position) {
      */
     public Mono<List<PositionTransaction>> upsertPositionTransactions(List<PositionTransaction> transactions,
```

Comment:
do we really need this upsertPositionTransactions method here? why not to call upsertTransactions directly?